src/tasks/templates.js

```diff
@@ -3,16 +3,27 @@ const { copyFiles } = require('mrm-core');
 
 // These files will be overwritten without any confirmation
 const files = [
-  '.travis.yml',
+  '.circleci/config.yml',
   'LICENSE',
   '.github/ISSUE_TEMPLATE.md',
   '.github/PULL_REQUEST_TEMPLATE.md',
+  '.prettierrc',
+  'appveyor.yml',
+  'eslint.js',
 ];
 
 // These files will by created only once
 const filesOnce = [
   'src/index.js',
   'src/cjs.js',
+  'src/options.json',
+  'test/errors.test.js',
+  'test/loader.test.js',
+  'test/options.test.js',
+  'test/fixtures/error.js',
+  'test/fixtures/fixtures.js',
+  'test/fixtures/foo.js',
```

Comment:
The default loader rule matches `test: /\.txt$/`, so maybe `test/fixtures/fixture.txt` instead with a few lines of `lorem` text